install.sh

```diff
@@ -104,17 +68,37 @@ else
   echo_info "Unable to detect system information, installation will continue."
 fi
 
+# Check if the user is on ChromeOS v117+ and not in the VT-2 console, sudo will not work
+if sudo | grep -q 'no new privileges'; then
```

Comment:
This line will just print the error to terminal and will always `false`, you need to redirect stderr with `2>&1`